test/core/vector/Vector_test.cpp

```diff
@@ -37,84 +37,81 @@ const char USAGE[] = "Usage:\n"
 
 sparta::app::DefaultValues DEFAULTS;
 
-class olympia::DecodeTester {
+class olympia::DecodeTester
```

Comment:
@AaronGChan I'd really like your input on my changes to the vector tester. I made sure to keep the original intent of the tests in place, but I had to make a lot of changes to get them to pass.

Reply:
I think your implementation is much cleaner! The json names would only need to be changed again if we add a lot more tests for varying vset values. In that case I could see it being necessary to include the vset values we're trying to set. 

Would it be useful at all to have a python script to autogenerate random vset tests to test edge cases? I think Knute and I had discussed this before to have a script for all vector instructions, so we could maybe expand the script as more functionality comes up. This is definitely more of a nice to have though and not pressing for getting vector support up.

Reply:
Good suggestion! I changed the names of the JSONs to include the vset config.

Yah I think auto-generating some tests would definitely be useful and would be easier to write in Python. Can you create an issue with your ideas?